pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
+    Copyright (c) 2017 Eurotech and/or its affiliates and others
```

Comment:
isn't thins supposed to be 2016-2017?

Reply:
Of course. I really don't recall of making such change, but I'll fix it anyway.